docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
     ulimits:
       nofile: 262144
   redis:
-    image: redis:3.2-alpine
+    image: redis:5.0-alpine
```

Comment:
Any specific reason for upgrading? Does this match what we run in prod/single tenant ?

Reply:
this matches what we run in CI